main/uwsgi/APKBUILD

```diff
@@ -9,14 +9,18 @@ pkgdesc="uWSGI application container server"
 url="http://projects.unbit.it/uwsgi/"
 arch="all"
 license="GPL2"
-depends="mailcap"
+depends="
+        mailcap
+        jansson
+        "
 install="uwsgi.pre-install"
 pkgusers="uwsgi"
 pkggroups="uwsgi"
 makedepends="
 	attr-dev
 	curl-dev
 	geoip-dev
+        jansson-dev
```

Comment:
As you can see, this file is indented with tabs, but you’ve used spaces here. Please fix it.